src/main/java/ladder/domain/RandomDecision.java

```diff
@@ -0,0 +1,11 @@
+package ladder.domain;
+
+import java.util.Random;
+
+public class RandomDecision implements BridgeDecision {
+    @Override
+    public boolean isBridgeTarge() {
+        Random random = new Random();
+        return random.nextInt(2) == 0 ? true : false;
```

Comment:
Random 객체는 한번만 생성해둬도 되지 않을까요? 🤔

```suggestion
        return random.nextBoolean();
```